features/Kiesrecht.feature

```diff
@@ -42,23 +105,78 @@ Scenario: de ingeschreven persoon is uitgesloten van kiesrecht
 Scenario: de ingeschreven persoon is niet uitgesloten van kiesrecht
 	Gegeven op de PL van een ingeschreven persoon is er geen categorie 13, element 38.10 (= persoon is niet uitgesloten)
 	Als de ingeschreven persoon wordt geraadpleegd
-	Dan is kiesrecht gelijk aan true
+	Dan is kiesrecht niet aanwezig

```

Comment:
De titel van de scenario in combinatie met de dan, is voor mij erg verwarrend. Moet ik, als consumer, als ik kiesrecht opvraag en het is niet aanwezig, dan concluderen dat de betreffende persoon niet is uitgesloten van kiesrecht of dat het kiesrecht onbekend is?

Reply:
Zoals ik het begrijp en in de feature bedoel zegt indicator kiesrecht niet iets over uitsluiting, maar over het hebben van kiesrecht. Wanneer iemand is uitgesloten van kiesrecht, weten we dat de persoon (op dat moment) geen kiesrecht heeft en dus zetten we kiesrecht = false. Wanneer iemand niet is uitgesloten van kiesrecht, weten we niets over het kiesrecht van de persoon. Het kiesrecht van de persoon is (voor de BRP) op dat moment onbekend/niet vastgesteld. In de API geven nemen we onbekend of niet vastgestelde waarden niet op, dus wordt element kiesrecht niet opgenomen.
De titel van het zegt iets over de situatie vanuit registratie-perspectief (vanuit de provider van de API):de persoon is uitgesloten. Het scenario beschrijft hoe dit in de api moet worden vertaald/weergegeven. Hoe zou jij dit scenario anders willen noemen dat het duidelijker is?

Reply:
Voor mij is het verwarrend omdat er geen scenario is waar kiesrecht gelijk is aan true

Reply:
Er is geen scenario waar kiesrecht is true, omdat we in BRP geen gegevens hebben waaruit we met zekerheid kunnen vaststellen dat iemand daadwerkelijk kiesrecht heeft. Dan wel we nu daar de kennis niet voor hebben. De criteria voor het hebben van kiesrecht zijn ook verschillend voor gemeenteraadsverkiezingen, provinciale verkiezingen en tweede kamer verkiezingen.
@MelvLee is de conclusie dan dat je de ontwerpkeuze voor indicator kiesrecht niet duidelijk vindt?

Reply:
@fsamwel Nee, eerder dat interpreteren of iemand kiesrecht heeft complex is en dat het niet makkelijk is om het te interpreteren uit het huidig model.